nategood/httpful/2024-05-01.yaml

```diff
@@ -0,0 +1,14 @@
+title:     Insecure HTTPS Connections due to Missing Default Certificate Validation
+link:      https://huntr.com/bounties/8d59c089-92f1-4b73-90f8-54968a70e2fb
+cve:       ~
+branches:
+    "0.1":
+        time:     2024-05-01 11:33:16
```

Comment:
If the issue isn't fixed in the 0.x branches. Shouldn't the value for time be `~`?

Reply:
I am not 100% about this, hence the question. Fortunately, there don't seem to be too many packages where security issues only get fixed in major releases and what I could find in past issues seems to be inconsistent e.g.  https://github.com/FriendsOfPHP/security-advisories/blob/e1e19e19d2d888dc9bbd8bd38cd793dc16577448/mautic/core/CVE-2021-27911.yaml and https://github.com/FriendsOfPHP/security-advisories/blob/e1e19e19d2d888dc9bbd8bd38cd793dc16577448/mediawiki/core/CVE-2019-12466.yaml